client/lib/plans/constants.js

```diff
@@ -161,18 +161,14 @@ export const TYPE_BUSINESS = 'TYPE_BUSINESS';
 
 const WPComGetBillingTimeframe = abtest => {
 	if ( abtest ) {
-		if ( isEnabled( 'upgrades/2-year-plans' ) && abtest( 'multiyearSubscriptions' ) === 'show' ) {
-			return i18n.translate( '/month, billed annually or every two years' );
-		}
+		return i18n.translate( '/month, billed annually or every two years' );
```

Comment:
Don't think this is good. We're defaulting to `billed annually or every two years`. And it doesn't matter if we pass the `abtest` var in here anymore.

We should probably get rid of the var, and default to a single translation here. :)

Reply:
@aidvu good catch, I updated that!